GitUI/Hotkey/HotkeySettingsManager.cs

```diff
@@ -290,6 +290,7 @@ public static HotkeySettings[] CreateDefaultSettings()
                     Hk(FormBrowse.Command.OpenWithDifftool, OpenWithDifftoolHotkey),
                     Hk(FormBrowse.Command.OpenWithDifftoolFirstToLocal, OpenWithDifftoolFirstToLocalHotkey),
                     Hk(FormBrowse.Command.OpenWithDifftoolSelectedToLocal, OpenWithDifftoolSelectedToLocalHotkey),
+                    Hk(FormBrowse.Command.OpenCommitsWithDifftool, Keys.None),
```

Comment:
You reminded me to keep the alphabetical order... Here you only almost did. 😉 It took me a moment to find it in the settings dialog.

Reply:
will change

Reply:
updated, rebased, squashed